src/components/common/SearchBarWizards.tsx

```diff
@@ -0,0 +1,132 @@
+import { useState, useRef } from "react";
```

Comment:
Since this is SearchBar specific to Wizards will all the logic in the file, think it should be moved to `src/components/ViewAgents/` or this component should be made generic so that we can use it everywhere

Reply:
Relocated it to ViewAgents, it can be made generic if its needed for anything else
https://github.com/elna-ai/ELNA-DApp/pull/204/commits/f968877a019c6292e10339cd2f2ecd9547e36978